### PR TITLE
Update the tracking budget page so the feature can be released.

### PR DESCRIPTION
**Pull request by youngcw**
Moved the page out of experimental and added a few minor things.

### CODE REVIEW

File: docs/getting-started/tracking-budget.md [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
+## What is the Tracking Budget
+
+The **Tracking budget** option allows the user to shift from the default **Envelope budget** option to a more traditional style of budgeting.
+The **Tracking budget** won't automatically rollover funds from month to month, and you don't have to account for all on budget funds continually.
+Instead of working with what funds you have _now_, you can forecast expenses and income.
+All transactions do still need to be categorized.
+
+We the Actual team suggest that you try to use the **Envelope Budget** if you haven't already.
+We believe it to be a more capable style of budgeting.
+If you still would prefer the more traditional **Tracking Budget** then you can switch to that if you would like.
```
Comment (matt-fidd) [inline docs/getting-started/tracking-budget.md:12]: Something read a bit strangely here, not sure if this is better though - what do you reckon?

```suggestion
We, the Actual team, suggest that you try to use the **Envelope Budget** if you haven't already as we believe it to be a more capable style of budgeting than the **Tracking Budget**. However, if you would prefer the more traditional style, then the **Tracking Budget** is for you.
```

**Review by matt-fidd — COMMENTED** [review]
LGTM, one small wording comment - let me know what you think

**Review by matt-fidd — APPROVED** [review]
Looks good to merge after it's released

File: docs/getting-started/tracking-budget.md [hunk 2]
```diff
@@ -58,14 +63,19 @@ Money does not automatically carry over from month to month in the **Tracking Bu
...
 If you utilize the **Rollover Overspending** feature on a category that is not over spent, the entire spent amount will be forwarded to the next month and the **Saved** amount will increase for the current month.
+This will affect your spent totals as if the spending didn't happen.
+The spending will only show up in the month that the rollover stops.
 :::
 
+<<<<<<< HEAD:docs/getting-started/tracking-budget.md
+## Other Features
+=======
 ## Working With the Budget
+>>>>>>> master:docs/experimental/tracking-budget.md
```
Comment (DavidHeiniluoma) [inline docs/getting-started/tracking-budget.md:74]: FYI looks like you have some merge conflict stuff that got left in

**Pull request merged**